tests/waveforms/cbc/test_cbc_waveforms.py

```diff
@@ -409,14 +434,14 @@ def test_phenom_p():
         hc_lal_data = hc_lal.data.data[lal_mask]
 
         assert np.allclose(
-            1e21 * hp_lal_data.real, 1e21 * hp_ml4gw.real.numpy(), atol=2e-3
+            1e21 * hp_lal_data.real, 1e21 * hp_ml4gw.real.numpy(), atol=1e-2
```

Comment:
Why was it necessary to increase the tolerance? Do we agree less closely with lalsim after these changes? Or are we testing a wider range of parameter space?

Reply:
The previous version of the waveform didn't match at all with lalsim when the tests were expanded to a larger parameter space. Now we match well in the full space, but certain parameters cause differences larger than ~6e-3, so I made it 1e-2.